packages/trader/src/AppV2/Components/TradeParameters/Multiplier/multiplier-wheel-picker.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import { ActionSheet, Text, WheelPicker, Skeleton } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useTraderStore } from 'Stores/useTraderStores';
+import debounce from 'lodash.debounce';
+
+type TMultiplierWheelPickerProps = {
+    multiplier: ReturnType<typeof useTraderStore>['multiplier'];
+    multiplier_range_list: ReturnType<typeof useTraderStore>['multiplier_range_list'];
+    currency: ReturnType<typeof useTraderStore>['currency'];
+    commission: ReturnType<typeof useTraderStore>['commission'];
+    setMultiplier: (multiplier: number) => void;
+};
+
+const debouncedSetMultiplier = debounce((setMultiplier, multiplier) => {
+    setMultiplier(multiplier);
+}, 200);
+
+const MultiplierWheelPicker = ({
+    multiplier,
+    multiplier_range_list = [],
+    currency,
+    commission,
+    setMultiplier,
+}: TMultiplierWheelPickerProps) => {
+    const multiplier_array = multiplier_range_list.map(item => ({ value: item.text }));
+    const initial_multiplier = React.useRef<number>();
+    const selected_multiplier = React.useRef<number>(multiplier);
+
+    useEffect(() => {
+        if (!initial_multiplier.current && multiplier) {
+            initial_multiplier.current = multiplier;
+        }
+        return () => {
+            if (initial_multiplier.current && initial_multiplier.current !== selected_multiplier.current) {
+                setMultiplier(initial_multiplier.current);
+            }
+            debouncedSetMultiplier.cancel();
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    const handlePickerValuesChange = (value: string | number) => {
+        const new_value = Number((value as string).slice(1));
```

Comment:
Sorry, why do you to copy value?

Reply:
For removing 'x'?

Reply:
so the wheel picker takes [x1, x2, x3]  bla bla bla. but onchange from trade store takes number 1, 2, 3. hence i slice the value because when i make change on wheel picker selected_value returns x1, x2, x3. So proposal the input has to be sliced

Reply:
hey in the next changes related to wheel-picker , we can send [{label:'x1', value:1}] as options to wheel picker and wheel picker will return the value. Basically label will be display text. I will update once the changes. After that we can remove this function and pass the input appropriately. Missed this in the initial changes

Reply:
label is optional so your current changes won't break, but we will improve this part